src/jennifer/adapter/postgres/command_interface.cr

```diff
@@ -44,7 +44,9 @@ module Jennifer
       end
 
       private def default_env_variables
-        {"PGPASSWORD" => config.password} of String => Command::Option
+        env = {"PGPASSWORD" => config.password} of String => Command::Option
+        env = env.merge({"PGPORT" => config.port.to_s}) unless config.port == -1
```

Comment:
I believe we can use here

```crystal
env["PGPORT"] = config.port.to_s unless config.port == -1
```

Reply:
Yes, that's correct, just updated it right now.